data/datasets/MNIST/test_MNISTDataset.py

```diff
@@ -1,11 +1,30 @@
-from MNISTDataset import MNISTDataset
 import matplotlib.pyplot as plt
+import pytest
+from MNISTDataset import MNISTDataset # use this import if running from Computer-Vision-TensorFlow/data/datasets/MNIST
+# from data.datasets.MNIST.MNISTDataset import MNISTDataset # use this import if running pytest from Computer-Vision-TensorFlow
+import tensorflow_datasets as tfds
+import tensorflow as tf
 
+# unit tests checking example from MNIST dataset for training asserting data shape, type, and load count
+def test_MNIST_dataset():
+    dataset = MNISTDataset(purpose='training')
+    assert(len(dataset.core) == 60000)
```

Comment:
you can add a string to print when assertion fails. e.g. `assert len(dataset.core) == 60000, f"{len(dataset.core)=}"`.